subsystem/src/main/resources/schema/elytron-tls-subsystem_1_0.xsd

```diff
@@ -0,0 +1,1336 @@
+<!--
+  ~ Copyright 2022 Red Hat, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
Just to confirm, at this point before the analysis doc review, there are no differences between the `tls` related resources here and the ones in the original subsystem, right?

Reply:
Some differences:

- TLS attributes that require security domains (ex. permission mapper, transformers) were removed
- the root level `register-jaspi-factory` attribute was removed
- commented out references to the future objects (ex. `key-store-object`)
- unimplemented base features are not yet included (filtering key stores, ldap key stores, server SSL SNI context, custom component configuration/attributes)
- A few features I hadn't looked into enough to know if I needed to implement them (JASPI, JACC, SASL, and jacc-policy/custom-policy)

Reply:
Please list the unimplemented base features that need to be added in the analysis doc so we don't forget about them. You can also list any other resources you're unsure about there so we can discuss them together.